cmd/promtool/main.go

```diff
@@ -151,6 +151,7 @@ func main() {
 	queryLabelsName := queryLabelsCmd.Arg("name", "Label name to provide label values for.").Required().String()
 	queryLabelsBegin := queryLabelsCmd.Flag("start", "Start time (RFC3339 or Unix timestamp).").String()
 	queryLabelsEnd := queryLabelsCmd.Flag("end", "End time (RFC3339 or Unix timestamp).").String()
+	queryLabelsMatch := querySeriesCmd.Flag("match", "Series selector. Can be specified multiple times.").Required().Strings()
```

Comment:
Needs to be `queryLabelsCmd` (this breaks as it is due to a duplicate flag), also drop Required() so we don't change the interface for existing users.

Reply:
Thanks for catching this error. Updated